src/main/java/edu/kis/powp/jobs2d/drivers/composite/IDriverComposite.java

```diff
@@ -0,0 +1,11 @@
+package edu.kis.powp.jobs2d.drivers.composite;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public interface IDriverComposite extends Job2dDriver {
+    void add(Job2dDriver driver);
+
+    void remove(Job2dDriver driver);
+
+    Job2dDriver getChildren(int index);
```

Comment:
The name seems inprecise (getChildren -> getChile). I am not sure that giving an access using index makes sense at the moment. The method is not used so maybe it is better to remove it. Maybe iterator would be more useful?